runusb

```diff
@@ -240,6 +244,10 @@ def main(args=sys.argv[1:]):
         # Bind-mount the actual path
         command.extend(('--bind', path))
 
+        # Attach a machine ID, use the path as the ID
+        print(get_machinename(path))
+        command.extend(('--machine', get_machinename(path)))
```

Comment:
If the `print` is intended to remain (and isn't a debug artefact), I'd prefer use of a variable to ensure these can't get out of sync. Else remove the `print`.